usaspending_api/awards/management/commands/helpers.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime
+
+from django.db.models import Q
+
+from usaspending_api.references.models import Location, RefCountryCode
+
+
+def up2colon(input_string):
+    'Takes the part of a string before `:`, if any.'
+
+    if input_string:
+        return input_string.split(':')[0].strip()
+    return ''
```

Comment:
loadcontracts and loadfinancialassistance use a lot of `row.get["something", ""]`, so at least in those cases, we've decided to save a `None` as a blank string instead of a `NULL` in the database.   I don't know which way we want to go, but we should probably be as consistent as we can.